components/server/ee/src/billing/billing-mode.ts

```diff
@@ -135,12 +135,12 @@ export class BillingModesImpl implements BillingModes {
         // 3. Check team memberships/plans
         // UBB overrides wins if there is _any_. But if there is none, use the existing Chargebee subscription.
         const teamsModes = await Promise.all(teams.map((t) => this.getBillingModeForTeam(t, now)));
-        const hasUbbTeam = teamsModes.some((tm) => tm.mode === "usage-based");
+        const hasUbbPaidTeamSeat = teamsModes.some((tm) => tm.mode === "usage-based" && !!tm.paid);
```

Comment:
Nit: Calling this a "seat" is slightly inaccurate (it's a "membership", where you're either a member of a team or you're not; it's not a "seat", because there is no "seat" belonging to a team which can be filled by someone or be left vacant) and thus confusing. However, fixing this is not important.

Reply:
Yeah, good point! Fill adjust.